api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/util/nameConventions.kt

```diff
@@ -19,11 +19,11 @@ internal fun defaultInitializerOpName(name: String) = "Init_$name"
 /** Default optimizer variable name in TensorFlow graph, based on variable's name. */
 internal fun defaultOptimizerVariableName(name: String) = "optimizer_$name"
 
-/** Default Conv2d bias variable name in TensorFlow graph, based on variable's name. */
-internal fun conv2dBiasVarName(name: String) = name + "_" + "conv2d_bias"
+/** Default Conv bias variable name in TensorFlow graph, based on variable's name. */
+internal fun convBiasVarName(name: String, dim: Int) = name + "_" + "conv${dim}d_bias"
 
-/** Default Conv2d kernel variable name in TensorFlow graph, based on variable's name. */
-internal fun conv2dKernelVarName(name: String) = name + "_" + "conv2d_kernel"
+/** Default Conv kernel variable name in TensorFlow graph, based on variable's name. */
+internal fun convKernelVarName(name: String, dim: Int) = name + "_" + "conv${dim}d_kernel"
```

Comment:
Need to ask: is it valid in Keras for conv1d and conv3d? did you check it?

Reply:
Tried to create an example but didn't find even `conv2d_kernel`. Could you write me what's wrong [here](https://colab.research.google.com/drive/1EhRfcgtGdb7FwhBIgKnCo095u0hKnO4j?usp=sharing)

Reply:
Sorry for that, it was the wrong question for you, ignore it.
All these names are used in KotlinDL only for graph manipulations and related to graph only and not related to the Keras imported config (excluding the layer name, of course).
I'm ok with this changes